compositor_render/src/transformations/web_renderer/chromium_sender.rs

```diff
@@ -45,10 +45,17 @@ impl ChromiumSender {
         }
     }
 
-    pub fn embed_sources(&self, sources: &[&NodeTexture]) -> Result<(), ChromiumSenderError> {
+    pub fn embed_sources(
+        &self,
+        sources: &[&NodeTexture],
+        children_ids: &[ComponentId],
```

Comment:
you are passing vector as a slice just to convert it back to the vector.

```suggestion
        children_ids: Vec<ComponentId>,
```